### ISSUE TITLE
pdf file locks nemo, maxing one cpu core

### ISSUE THREAD

**Issue by dirtcooker** [linuxmint/nemo#2255]
```
 * Nemo version (nemo --version) 4.2.3
 * Is issue with desktop or windowed nemo? both
[two pdfs lock up nemo.tar.gz](https://github.com/linuxmint/nemo/files/3935484/two.pdfs.lock.up.nemo.tar.gz)

 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) 18.3, 19.2
 * Graphics hardware *and* driver used GeForce GTX 960M, 435.21, other machines as well
 * 32 or 64 bit 64 bit
 ```

**Issue**
the attached archive contains 2 pdf files, which when extracted to the hard drive will lock up nemo.

**Steps to reproduce**
extract files, then open the directory with nemo. 
Thumbnails are off. list view is on.

**Expected behaviour**
one should be able to see the files in the directory. However, the file names are not listed, nemo is unresponsive and maxing one cpu core. On closing nemo there is a delay, then Force Quit to terminate.

**Other information**
open the tar archive, double-click on a file, it opens properly in the viewer.

**Comment by smurphos** [comment]
I can't reproduce this. The files show up fine in nemo once extracted. No issue opening them. Tested with 4.2.3 and 4.4.1

**Comment by jaheme127** [comment]
@leigh123linux  
you are right, the hanging is caused by the pdf files. I reproduced it. So it seems to be the same issue. 

Nontheless, it would be nixe to check that before closing the ticket if it's really the cause. 

Thank you for your effort helping to get Nemo greater ;-)

**Comment by jaheme127** [comment]
I reproduced the issue with the attached files and could isolate the issue to one of my PDF showing my IKEA family card.

Please note, that if I wait a several amount of time the pdf preview as well as the file list is in comparision to @dirtcooker shown correctly

I will try to change my plugin layout to test if there is a change in the blocking behavior

**Comment by jaheme127** [comment]
In investigated this issue further and came to the conclusion that this is not an issue in nemo, but it comes up when you activate the plugin: nemo-media-columns.

I would presume that there is an issue with calculation the content of the media columns on certain pdf files. -  can this ticket be moved to nemo-extensions?

**Comment by dirtcooker** [comment]
Yes! I disabled nemo-media-columns and the problem has gone away. Good work isolating the problem.
